### ISSUE TITLE
[ALBERT] Pretraining

### ISSUE THREAD

**Issue by stefan-it** [google-research/google-research#74]
Hi,

thanks for releasing implementation and pre-trained models for ALBERT :heart: 

I would really like to train a model from scratch - would this be possible with one v3-8 TPU 🤔. If that's possible could you also give a detailed overview of the parameters you used for training base or larger models?

I think the "training for x steps for sequence length 128 and then fine-tuning the model for a sequence length of 512" approach is not used in the paper, so could you specify the parameters for the `create_pretraining_data.py` script (because it is using a seq. length of 128 by default).

Thanks many in advance!

**Comment by lanzhzh** [comment]
Hi Hi, 

Sorry for the delay, we didn't get notification about these issues. We are moving our code to a separate repo so we can handle your questions in a more timely way. 

I did update the setting to be our default setting about 3 days ago. I am not sure what v3-8 is, but you can try it on the albert-base. Please do turn off the dropout, which could save you a significant portion of memory.  You can also try to lower your batch size. For example, using the 512 setting here (https://arxiv.org/abs/1904.00962).
